core/loop.gms

```diff
@@ -71,9 +71,12 @@ if( (cm_startyear gt 2005),
 ***--------------------------------------------------------------------------
 ***         SOLVE
 ***--------------------------------------------------------------------------
-***this disables solprint in cm_nash_mode=debug case by default. It is switched on in case of infes in nash/solve.gms
-*RP* for faster debugging, turn solprint immediately on
-$IF %cm_nash_mode% == "debug" option solprint = on ;
+*** This disables solprint in cm_nash_mode=1 (debug) case by default. It is switched on in case of infes in nash/solve.gms
```

Comment:
I actually don't get the comment. Setting solprint to `on` disables it by default?

Reply:
Lazy commenting.
```diff
diff --git a/core/loop.gms b/core/loop.gms
index a7758ac..d57b3f7 100644
--- a/core/loop.gms
+++ b/core/loop.gms
@@ -63,7 +63,8 @@ if( (cm_startyear gt 2005),
 ***         SOLVE 
 ***--------------------------------------------------------------------------
 ***this disables solprint in cm_nash_mode=debug case by default. It is switched on in case of infes in nash/solve.gms
-$IF %cm_nash_mode% == "debug" option solprint = off ;
+*RP* for faster debugging, turn solprint immediately on
+$IF %cm_nash_mode% == "debug" option solprint = on ;
 
 o_modelstat = 100;
 loop(sol_itr$(sol_itr.val <= c_solver_try_max),
```
(You will have to look on the [Gitlab repo](https://gitlab.pik-potsdam.de/REMIND/REMIND/-/commit/3acd28ce).)